src/util/sync/sync_spinlock.h

```diff
@@ -21,7 +21,11 @@ namespace dxvk::sync {
   void spin(uint32_t spinCount, const Fn& fn) {
     while (unlikely(!fn())) {
       for (uint32_t i = 1; i < spinCount; i++) {
+#if defined(CPU_ARCH_X86)
         _mm_pause();
+#elif defined(CPU_ARCH_ARM)
+        __asm__ __volatile__ ("yield");
```

Comment:
Is there an intrinsic for this?

Reply:
Looked up GCC intrinsic but there is none for ARM/64 :( apparently only MSVC has "__yield()"...

Also after the loop it calls dxvk::this_thread::yield(); which calls stdlib yield, isn't already doing same as the intrinsic/asm instructions under the hood? I wasn't sure anyway so I just searched the x86 pause equivalent for ARMs and came with yield